chat-client/src/client/mynahUi.ts

```diff
@@ -200,10 +201,11 @@ export const createMynahUi = (
         },
         onTabAdd: (tabId: string) => {
             const defaultTabBarData = tabFactory.getDefaultTabData()
+            const contextCommands = [...(contextCommandGroups || []), ...(featureConfig?.get('contextCommands') || [])]
```

Comment:
Why are we doing here some thing different in comparison to what was done above? Can we share the logic? Why is 'contextCommands' key used here, but not in the logic above?